src/main.rs

```diff
@@ -1,5 +1,9 @@
 use std::io;
 
+fn count_overlap(secret: &str, guess: &str) -> usize {
+    secret.chars().filter(|letter| guess.contains(*letter)).count()
```

Comment:
@ABaldwinHunter in fact it's less performant, by 6 nanoseconds https://github.com/ABaldwinHunter/gelato/pull/2/commits/42a258d38bd41fe2e2cfa9769c73ccf23b3d819b 🤔